src/blueprint-ast-to-runtime.coffee

```diff
@@ -57,7 +58,12 @@ blueprintAstToRuntime = (blueprintAst, filename) ->
           }
 
         # expand URI parameters
-        uriResult = expandUriTemplateWithParameters resource['uriTemplate'], parameters
+        if action.attributes?.uriTemplate? and action.attributes?.uriTemplate != ''
```

Comment:
this could be easily written as `if action.attributes.uriTemplate`, or am I missing any specialities here?
Probably `"0"`?